mrmustard/physics/triples.py

```diff
@@ -447,3 +447,37 @@ def fock_damping_Abc(n_modes: int) -> Union[Matrix, Vector, Scalar]:
     c = 1.0 + 0j
 
     return A, b, c
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Maps between representations
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+def displacement_gate_s_parametrized_Abc(s: int) -> Union[Matrix, Vector, Scalar]:
+    r"""
+    The ``(A, b, c)`` triple of a single-mode ``s``-parametrized dispalcement gate ``D(\gamma)``.
+    Given the complex variables for this single-mode is ``(z^*, z)`` corresponding to [out_ket, in_ket] unitary ordering,
```

Comment:
Can you use :math: for things like ``z^*``? Otherwise, they will render as you see them in this comment